plugins/node.d.midnightbsd/if_

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
+#
+# Wildcard-plugin to monitor network interfaces. To monitor an
+# interface, link if_<interface> to this file. E.g.
+#
+#    ln -s /usr/share/munin/node/plugins-auto/if_ /etc/munin/node.d/if_eth0
+#
+# ...will monitor eth0.
+#
+# To aggregate all network interfaces on the system (except lo0),
+# link if_aggregated to this file.
+#
+# Any device found in /usr/bin/netstat can be monitored.
+#
+# Magic markers (optional - used by munin-config and some installation
+# scripts):
+#
+#%# family=auto
+#%# capabilities=autoconf suggest
+
+INTERFACE=${0##*if_}
+
+if [ "$1" = "autoconf" ]; then
+	if [ -x /sbin/ifconfig -o -x /usr/bin/netstat ]; then
```

Comment:
`shellcheck` would appreciate `[ ... ] || [ ... ]` instead of `-o` (order of execution is better defined).